game-core/src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -145,16 +142,14 @@ private void addDisplayPlayersInformationMenu(final JMenu parentMenu) {
   }
 
   private void addBanUsernameMenuItem(final JMenu parentMenu) {
-    final JMenuItem menuItem = new JMenuItem("Ban Username");
+    final JMenuItem menuItem = new JMenuItem("Blacklist Player Name");
```

Comment:
What's the thinking on changing 'Username' to 'Player Name'?

Reply:
I might have over-thought it. The text was updated to try and help avoid any confusion over the word 'ban' and if a 'username' was a player or if we were really banning the players name.  Second, having 'Ban Player' and 'Ban Player Name' in the same list next to each other seemed too similar, maybe potential for misclick.

Reply:
Changing to 'Blacklist' I think you can make a good argument for since these changes are really moving in the direction of treating 'usernames' (blacklist) differently then 'IP/MAC' (ban). But my initial reaction is its better to stick with 'Username' and stay consistent across the UI and code.

Reply:
Hmm, previously 'username' was synonymous with user identity. I'm not sure it was really clear that a 'Ban username' was banning someones name and not them. I don't know if I made that point clear enough. I do suppose it is fine to keep the terminology more consistent for at least now.

Reply:
Updated in: 0207951

Reply:
Ok. There are a decent number of other places you put in "player name" or "playerName" into UI text or code. Can you do a quick search and replace on those so that everything is just "username"?